client/app/dbaas/logs/detail/inputs/home/logs-inputs-home.controller.js

```diff
@@ -0,0 +1,194 @@
+     * @memberof LogsInputsCtrl
+     */
+    info (input) {
+        this.CloudMessage.flushChildMessage();
+        this.ControllerHelper.modal.showModal({
+            modalConfig: {
+                templateUrl: "app/dbaas/logs/detail/inputs/home/info/logs-inputs-home-info.html",
+                controller: "LogsInputsHomeInfoModalCtrl",
+                controllerAs: "ctrl",
+                resolve: {
+                    currentInput: () => input
+                }
+            }
+        });
+    }
+
+    /**
+     * Shows the confirmation modal box for input deletion confirmation
+     * and deletes the input if the user confirms the deletion
+     *
+     * @param {any} input - the input object
+     * @memberof LogsInputsCtrl
+     */
+    showDeleteConfirm (input) {
+        this.CloudMessage.flushChildMessage();
+        return this.ControllerHelper.modal.showDeleteModal({
+            titleText: this.$translate.instant("inputs_delete"),
+            text: this.$translate.instant("inputs_delete_message", { input: input.info.title })
+        }).then(() => this._delete(input));
+    }
+
+    /**
+     * Restarts the input
+     *
+     * @param {any} input - the input to be restarted
+     * @memberof LogsInputsCtrl
+     */
+    restartInput (input) {
+        this.CloudMessage.flushChildMessage();
+        this.CloudMessage.info(this.$translate.instant("inputs_restarting", { inputTitle: input.info.title }));
+        this._setInputToProcessing(input);
+        this.processInput = this.ControllerHelper.request.getHashLoader({
+            loaderFunction: () => this.LogsInputsService.restartInput(this.serviceName, input)
+        });
+        this.processInput.load()
+            .then(() => this._reloadInputDetail(input.info.inputId),
+                  () => this._reloadInputDetail(input.info.inputId));
+    }
+
+    /**
+     * navigates to the standard output page
+     *
+     * @param {any} input - the input for which standard output is to be displayed
+     * @memberof LogsInputsCtrl
+     */
+    standardOutput (input) {
+        this.CloudMessage.flushChildMessage();
+        this.$state.go("dbaas.logs.detail.inputs.console", {
+            serviceName: this.serviceName,
+            inputId: input.info.inputId
+        });
+    }
+
+    /**
+     * Starts the input
+     *
+     * @param {any} input - the input to be started
+     * @memberof LogsInputsCtrl
+     */
+    startInput (input) {
+        this.CloudMessage.flushChildMessage();
+        this.CloudMessage.info(this.$translate.instant("inputs_starting", { inputTitle: input.info.title }));
+        this._setInputToProcessing(input);
+        this.processInput = this.ControllerHelper.request.getHashLoader({
+            loaderFunction: () => this.LogsInputsService.startInput(this.serviceName, input)
+        });
+        this.processInput.load()
+            .then(() => this._reloadInputDetail(input.info.inputId),
```

Comment:
remove that if you make the change above

Reply:
This is done as above